# request
sensor_msgs/Image rgb
sensor_msgs/Image prediction
sensor_msgs/Image corners
sensor_msgs/Image outer_edges
sensor_msgs/Image inner_edges
---
# response
int64 py
int64 px
float64 angle
int64 inner_py
int64 inner_px
float64[] var
float64[] angle_x
float64[] angle_y
